others/Reverse_spiral_form_of _matrix.cpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file
+ * @brief Given a 2D array print it in reverse spiral form
+ */
+#include <iostream>
+#include <stdlib.h>
+#include <algorithm>
+#include <vector>
+
+using namespace std;
+vector<int> reverseSpiral(int R, int C, vector<vector<int>> &a)
```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.